docker/Dockerfile

```
@@ -53,8 +53,10 @@ RUN apt-get update \
       awscli
 
 # Allow running datadog-ci in CI with npx
-RUN apt-get install -y --no-install-recommends nodejs npm \
-  && npm install -g @datadog/datadog-ci
+RUN curl -fsSL https://deb.nodesource.com/setup_20.x | bash - \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends nodejs npm \
+    && npm install -g @datadog/datadog-ci
```

```suggestion
RUN  apt-get install -y --no-install-recommends nodejs npm \
    && npm install -g @datadog/datadog-ci
```
Actually, with the switch to bookworm, we don't need to install from nodesource anyway.